### ISSUE TITLE
cluster_advertise documentation inconsistent

### ISSUE THREAD

**Issue by jessecollier** [Kong/kong#2164]
### Summary

Running with `cluster_advertise` per the documentation on the website leads to errors:

```bash
+ KONG_CLUSTER_ADVERTISE=10.100.3.157
+ exec kong start
Error: cluster_advertise must be in the form of IPv4:port
```

Current documentation does not mention port:
```
cluster_advertise

By default, the cluster_listen address is advertised over the cluster.
 If the cluster_listen host is '0.0.0.0', then the first local, non-loopback IPv4 address
 will be advertised to other nodes. However, in some cases (specifically NAT traversal),
 there may be a routable address that cannot be bound to. This flag enables advertising a
 different address to support this.
```

### Steps To Reproduce

1. Run kong with `cluster_advertise = x.x.x.x` without port

I would recommend giving either an example, or atleast specifying the port requirement here.

IMO it would preferable to default to same as listen port, making it optional to specify. The reason for this is because I am dynamically getting the advertise IP. Because the port is part of another variable, I would have to parse that as well to match the port.

**Comment by thefosk** [comment]
@jessecollier that's definitely a mistake in the doc.  Will update.

**Comment by jhenry82** [comment]
There's also either a second bug in the cluster_advertise docs, or the setting's behavior, in 0.10. See my comment in https://github.com/Mashape/kong/issues/2182. But basically with cluster_listen = 0.0.0.0, kong is advertising itself as 127.0.0.1 even though my VM has a 10.x.x.x IP available to use. This same config worked correctly in 0.9.8.

If I force cluster_advertise to use my 10.x IP, it works, so there's at least an easy workaround.

**Comment by thibaultcha** [comment]
Hi,

This property has disappeared with the release of Kong 0.11.0, as our Serf dependency has been removed and clustering capabilities are now natively implemented in Kong.

@jhenry82 I believe I have seen this issue before, and the reason behind it was an change in Serf itself, which we bumped between Kong 0.10 and Kong 0.9... Nothing under our control here, sorry about that!

Thank you for the report!
